src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
This PR is building on top of @dagood  work with #646. $646 enables the scenario where you want to get the latest versions for what is specified in the DEFAULT test runtime project.json. It does not allow us to change to a different location. We need both default and a different project.json for both minimum dependencies and the latest dependencies. My change is to enable it points to a different location.

As for the build failures, it's because the validation rule check all project.jason file versions according to  what is defined in the dirs.prop.

@dagood we are discussing a non default jason file under src directory, we use it to test latest dependencies. All version in this file will be changed into * in the build time and copy to the bin directory.